src/config/cron.md

```diff
@@ -1,13 +1,28 @@
 # Cron
 
-Void Linux comes without a default cron daemon, you can choose one of multiple
-cron implementations, by installing the package and enabling the system service.
+[cron](https://en.wikipedia.org/wiki/Cron) is a daemon for running programs at
+regular intervals. The programs and intervals are specified by using
+[crontab(1)](https://man.voidlinux.org/crontab.1) to edit a `crontab` file.
+`crontab` files can be system-wide or user-specific; to edit the system crontab,
+run `crontab` as the superuser:
 
-Available choices include [cronie](https://github.com/cronie-crond/cronie/),
+```
+# crontab -e
+```
+
+Similarly, run `crontab -e` under your own login to edit your personal crontab.
+
+By default, a cron daemon is not installed. However, multiple cron
+implementations are available, including
+[cronie](https://github.com/cronie-crond/cronie/),
 [dcron](http://www.jimpryor.net/linux/dcron.html),
 [fcron](http://fcron.free.fr/) and more.
 
-As alternative to the standard cron implementations you can use something like
-[snooze](https://github.com/leahneukirchen/snooze) or
-[runwhen](https://code.dogmap.org/runwhen/) which go hand in hand with service
-supervision.
+Once you have chosen and installed an implementation,
+[enable](./services/index.md#enabling-services) the `crond` service.
+Implementation documentation will be available in
+[crond(8)](https://man.voidlinux.org/crond.8).
+
+As an alternative to the standard cron implementations, you can use things like
+[snooze(1)](https://man.voidlinux.org/snooze.1), which go hand-in-hand with
+service supervision.
```

Comment:
Why remove the `runwhen` mention? Perhaps add that the `snooze` package comes with `snooze-{hourly,daily,weekly,monthly}` services?

Reply:
Because `runwhen` doesn't seem to be in the repos. Are we okay to suggest software that's not packaged?

Reply:
Oh, and sure, will mention those things about `snooze`.

Reply:
Lol, definitely not. Good catch!

Reply:
:-) Still, i'll update the commit message to explain why `runwhen` is no longer mentioned (which i should have done in the first place).

Reply:
i've now mentioned the `snooze-*` services, listing them individually because i couldn't find a way to use the abbreviated (non-sh) shell-syntax version without it feeling awkward.